### PR TITLE
fix #1316: Fix incorrect icon on fetch profile error

### LINKED ISSUE #1316: Incorrect icon shown
**Summary:**

When user tries to open user profile in absence of internet connection, the message shown contains incorrect icon.

**Steps to reproduce:**

* Login into the app
* Turn off mobile data/Wifi
* Open user profile 
* See the screen being displayed

**Expected behaviour:**

The icon should be replaced with an appropriate one

**Screenshots:**
WhatsApp Image 2020-02-05 at 4 50 05 PM

### PR DESCRIPTION
**Pull request by Sparsh1212**
Fixes #1316 

Screenshots:

1. **Old icon:**

2. **New icon:**

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Sparsh1212** [comment]
@miPlodder @devansh-299 Please review.

File: app/src/main/java/org/mifos/mobile/ui/fragments/UserProfileFragment.kt [hunk 1]
```diff
@@ -256,7 +256,7 @@ class UserProfileFragment : BaseFragment(), UserDetailsView {
     override fun showError(message: String?) {
         Toaster.show(rootView, message)
         sweetUIErrorHandler?.showSweetCustomErrorUI(getString(R.string.error_fetching_user_profile),
-                R.drawable.ic_assignment_turned_in_black_24dp, appBarLayout,
+                R.drawable.ic_assignment_not_turned_black_24dp, appBarLayout,
```
Comment (miPlodder) [inline app/src/main/java/org/mifos/mobile/ui/fragments/UserProfileFragment.kt]: can we use any existing drawable? I guess it should be present, check other fragments when no internet is present and confirm

**Review by miPlodder — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/mifos/mobile/ui/fragments/UserProfileFragment.kt (hunk 1 above)
Comment (Sparsh1212) [reply app/src/main/java/org/mifos/mobile/ui/fragments/UserProfileFragment.kt]: Does this looks appropriate ?. I have used an existing drawable for this.

Comment (Sparsh1212) [reply app/src/main/java/org/mifos/mobile/ui/fragments/UserProfileFragment.kt]: @miPlodder Please review the updated changes.

**Comment by miPlodder** [comment]
Good

**Pull request merged**